scheds/rust/scx_layered/src/bpf/main.bpf.c

```diff
@@ -1768,6 +1768,11 @@ static __noinline bool match_one(struct layer_match *match,
 		bpf_rcu_read_unlock();
 		return nsid == match->nsid;
 	}
+	case MATCH_PID_TGID_EQUALS: {
+		// There is nuance to this around exec(2)s and group leader swaps.
+		// See https://github.com/sched-ext/scx/issues/610 for more details.
+		return p->tgid == p->pid && match->pid_tgid_eq;
```

Comment:
I wonder whether the better name for this match is something like `is_group_leader`.